examples/2-query/src/Container.re

```diff
@@ -11,32 +11,35 @@ module GetAllPokemons = [%graphql
 |}
 ];
 
-let request = GetAllPokemons.make(~first=151, ());
-
-let flattenPokemon = pokemons =>
-  pokemons
-  |> Array.map(pokemon =>
-       switch (pokemon) {
-       | Some(pokemon) => Belt.Option.getWithDefault(pokemon##name, "")
-       | None => ""
-       }
-     );
+let flattenPokemon = pokemons => {
+  GetAllPokemons.(
+    pokemons->Belt.Array.map(pokemon =>
+      pokemon
+      ->Belt.Option.flatMap(p => p.name)
+      ->Belt.Option.getWithDefault("")
+    )
+  );
+};
 
 [@react.component]
 let make = () => {
   let (Hooks.{response}, _) =
-    Hooks.useQuery(~request, ~requestPolicy=`CacheFirst, ());
+    Hooks.useQuery(
+      ~query=(module GetAllPokemons),
+      ~requestPolicy=`CacheFirst,
+      GetAllPokemons.makeVariables(~first=151, ()),
```

Comment:
Nit – I'd prefer our examples to use record syntax rather than `makeVariables` for supplying variables to hooks:

```suggestion
      GetAllPokemons.{ first: 151 },
```

Reply:
Are you sure a local open is needed here. I don't think it's needed in the Apollo bindings in this scenario.